client/src/js/main.js

```diff
@@ -11,7 +11,7 @@ const calculatorFormInput = calculatorForm.querySelector("input[name='calculator
 calculatorClearButton.addEventListener("click", removeLastCalculatorSymbol)
 calculatorFormInput.addEventListener("input",  () => calculatorForm.classList.remove("error"))
 
-const ws = new WebSocket("ws://0.0.0.0:5000"); // it seems like for virtual ip = 10.129.0.15 but it starts endless connecting
+const ws = new WebSocket("ws://84.201.143.213:5000"); // it seems like for virtual ip = 10.129.0.15 but it starts endless connecting
```

Comment:
I think that we should not add such values to code (move either to some file with secrets or to some file with configuration)